File: Python/7kyu/VovewlCount.py
import re


def getCount(sentence):
    return len([v for v in sentence if 'aeiou'.find(v) >= 0])


def getCount_1(sentence):
    return sum(1 for w in sentence if w in 'aeiou')


def getCount_2(sentence):
    return sum(c in 'aeiou' for c in sentence)

File: JavaScript/7kyu/VovewlCount.js
function getCount(str) {
  return (str.match(/[aeiou]/g) || []).length;
}

function getCount_1(str) {
  return str
    .split()
    .map((s) => "aeiou".includes(s))
    .join().length;
}

const getCount_2 = (str) => str.replace(/[^aeiou]/g, "").length;

// var r = getCount_2("abracuadabra");
// console.log(r);

export { getCount, getCount_1, getCount_2 };
